neon_core/__init__.py

```diff
@@ -119,6 +119,8 @@ def setup_ovos_config():
 from neon_core.skills import NeonSkill, NeonFallbackSkill
 from neon_core.skills.intent_service import NeonIntentService
 
+from neon_utils.signal_utils import init_signal_handlers
+init_signal_handlers()
```

Comment:
not sure about the best location for this one, bus might not have been started yet or we might be writing some standalon scripts or tests...

i am not sure on the best place, but in general i would avoid doing it on module import. maybe in the individual services main entrypoints?

i see this pretty much as the call to init locale which is done this way

Reply:
That makes sense.. I put it here to be with all the other OVOS pieces, but it probably should go directly in the skill module init which is also consistent with the other modules